src/viewport/api/public.py

```
@@ -165,25 +206,32 @@ async def _build_public_gallery_response(
             )
             cover_photo_obj = fallback_photos[0] if fallback_photos else None
 
     # Override cover with project-level cover when provided (for project folder views)
-    effective_cover: PublicCover | None = None
+    effective_cover: MediaCover | None = None
     if override_cover is not None:
         effective_cover = override_cover
     else:
-        if cover_photo_obj and cover_photo_obj.object_key and cover_photo_obj.thumbnail_object_key:
-            cover_full_url = full_url_map.get(cover_photo_obj.object_key)
+        if cover_photo_obj and cover_photo_obj.thumbnail_object_key:
```

Ruff (0.15.20)

[warning] Use `elif` instead of `else` then `if`, to reduce indentation

Convert to `elif`

(PLR5501)